<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editproducts</class>
 <widget class="QDialog" name="editproducts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>761</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>My App</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editpro_mainpagebutton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>MainPage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editpro_searchbutton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editpro_editbutton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>440</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editpro_deletebutton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>in licenses</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editpro_search1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editpro_search2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editpro_search3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>741</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>721</width>
     <height>71</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Factory</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Category</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>License</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>DisCount%</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Factory :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Category :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>License :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Price :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>460</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DisCount :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>420</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>$</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editpro_edit_name">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editpro_edit_fac">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editpro_edit_quan">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editpro_edit_pric">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="editpro_edit_disc">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>20</width>
     <height>321</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="licenstext">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="editpro_edit_categ">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-----</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Drink</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Snack</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Food</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dairy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Others</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>License :</string>
   </property>
  </widget>
  <widget class="QLabel" name="licenstext_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>230</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>editpro_search1</tabstop>
  <tabstop>editpro_search2</tabstop>
  <tabstop>editpro_search3</tabstop>
  <tabstop>editpro_searchbutton</tabstop>
  <tabstop>editpro_edit_name</tabstop>
  <tabstop>editpro_edit_fac</tabstop>
  <tabstop>editpro_edit_categ</tabstop>
  <tabstop>editpro_edit_quan</tabstop>
  <tabstop>editpro_edit_pric</tabstop>
  <tabstop>editpro_edit_disc</tabstop>
  <tabstop>editpro_editbutton</tabstop>
  <tabstop>editpro_deletebutton</tabstop>
  <tabstop>editpro_mainpagebutton</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
